charts/openproject/values.yaml

```
@@ -419,6 +415,9 @@ hocuspocus:
 
   podAnnotations:
 
+  allowedOpenProjectDomains:
+    - openproject.com
```

While this is the correct value for our own SaaS deployment, this helm chart is for everyone and we are so far not including real values here.

Looking at the hostname that we define [for OP in this file already](https://github.com/opf/helm-charts/blob/7a2f001e25e0e43bec538306a4be3309be3c3358/charts/openproject/values.yaml#L144), this should probably be

```suggestion
    - openproject.example.com
```

---

Pushed a separate update since I needed to also update the spec :+1: